Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -9,6 +9,19 @@ public enum ViewAnnotationManagerError: Error {
     case geometryFieldMissing
 }
 
+/// Represents an annotation view.
+public struct ViewAnnotation {
```

Comment:
It seems to me that introducing a whole new entity is a bit overkill for this case, would `Dictionary< ViewAnnotationOptions, UIView>` do the trick here?

Reply:
> Clients need to wrap crud methods and keep the list on their side if they want to iterate over view annotations and perform some action.

If I understand the task correctly, this is to move this from client to our code base, keep the list of added view and its options. If we do a dictionary like suggested then again the client would need to keep a list of options on their side.

Reply:
With `annotations` defined like this: `public var annotations: Dictionary<ViewAnnotationOptions, UIView>`, `annotations.keys` returns the list of options and `annotations.values` returns the list of views.